lnbits/core/crud.py

```diff
@@ -454,15 +454,24 @@ async def update_payment_details(
 async def update_payment_extra(
     payment_hash: str,
     extra: dict,
+    outgoing: bool = False,
+    incoming: bool = False,
     conn: Optional[Connection] = None,
 ) -> None:
     """
     Only update the `extra` field for the payment.
     Old values in the `extra` JSON object will be kept unless the new `extra` overwrites them.
     """
+    amount_clause = ""
+
+    if outgoing != incoming:
+        if outgoing:
+            amount_clause = "AND amount < 0"
+        else:
+            amount_clause = "AND amount > 0"
```

Comment:
I would go for default `outgoing=True` and drop the `incoming` flag altogether.

Reply:
done, defaults to `outgoing=False`